app/android/src/main/java/com/capacitorjs/plugins/app/AppPlugin.java

```diff
@@ -73,7 +74,13 @@ public void exitApp(PluginCall call) {
     public void getInfo(PluginCall call) {
         JSObject data = new JSObject();
         try {
-            PackageInfo pinfo = getContext().getPackageManager().getPackageInfo(getContext().getPackageName(), 0);
+            PackageInfo pinfo = null;
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.TIRAMISU) {
```

Comment:
```suggestion
            if (Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.TIRAMISU) {
```

You can use it like this because we have the `import android.os.Build;` that was unused.
Or if you prefer to keep the `android.os.Build.VERSION.SDK_INT`, then can you remove the unused import?